plugins/kubernetes/app/models/kubernetes/release.rb

```diff
@@ -97,8 +97,8 @@ def fetch_pods
             release_id: id
           }.to_kuber_selector
         }
```

Comment:
thx, I was wondering why this never returned anything ... did not do the 10gig test :/